depth_image_topic: /XTION/depth/image
camera_info_topic: /XTION/depth/camera_info
#source: /media/My Passport/dataset
#source: /home/mwuethri_local/data/complete_projects/coordinate_particle_filter/humanoids.2014/evaluation/moving_boxes/sim/dataset
source: camera


max_kl_divergence: 2.0

use_gpu: false
# # # evaluation_count: 100 #the number of evaluations
# # # 
# # # # filtering parameters ----------------------------------------------------------------------
# # # downsampling_factor: 8 #8
# # # initial_sample_count: 1000 #10000 this is the number of samples for the initialization
# # # max_sample_count: 10000 #10000 this is for the gpu to allocate memory
# # # 
# # # # process model parameters ------------------------------------------------------------------
# # # #linear_acceleration_sigma:   2.0 #0.1  0.2  1.0 2.0
# # # #angular_acceleration_sigma: 30.0 #1.5  3.0  15.0 30.0
# # # #damping: 1000.0 #5.0  20.0  1000.0 1000.0
# # # 
# # # linear_acceleration_sigma:   0.1 #0.1  0.2  1.0 2.0
# # # angular_acceleration_sigma: 1.5 #1.5  3.0  15.0 30.0
# # # damping: 20.0 #5.0  20.0  1000.0 1000.0
# # # 
# # # # observation model parameters --------------------------------------------------------------
# # # sensor_failure_probability: 0.0
# # # object_model_sigma: 0.005
# # # sigma_factor: 0.00142478
# # # half_life_depth: 1.0
# # # max_depth: 6.0
# # # min_depth: 0.0
# # # 
# # # # # occlusion process model stuff
# # # # # initial prob of the source being the object for each pixels
# # # # initial_occlusion_prob: 0.1 #0.1
# # # # # prob of source being object given that one second ago the source was the object
# # # # p_occluded_visible: 0.1 #0.1
# # # # # prob of source being object given that one second ago source was not object
# # # # p_occluded_occluded: 0.7 #0.7
# # # # occlusion_process_sigam: 0.2
# # # 
# # # 
# # # # 
# # # # occlusion process model stuff
# # # # initial prob of the source being the object for each pixels
# # # initial_occlusion_prob: 0.0 #0.1
# # # # prob of source being object given that one second ago the source was the object
# # # p_occluded_visible: 0.0 #0.1
# # # # prob of source being object given that one second ago source was not object
# # # p_occluded_occluded: 0.0 #0.7
# # # occlusion_process_sigam: 0.00000001

evaluation_count: 100 #the number of evaluations

# filtering parameters ----------------------------------------------------------------------
downsampling_factor: 8 #8
initial_sample_count: 1000 #10000 this is the number of samples for the initialization
max_sample_count: 10000 #10000 this is for the gpu to allocate memory

linear_acceleration_sigma:   0.1 #0.1  0.2  1.0 2.0
angular_acceleration_sigma: 1.5 #1.5  3.0  15.0 30.0
damping: 5.0 #5.0  20.0  1000.0 1000.0

# occlusion process model stuff
# initial prob of the source being the object for each pixels
initial_occlusion_prob: 0.0 #0.1
# prob of source being object given that one second ago the source was the object
p_occluded_visible: 0.1 #0.1
# prob of source being object given that one second ago source was not object
p_occluded_occluded: 0.2 #0.7
# continuous occlusion process model
occlusion_process_sigma: 0.1

# observation model parameters --------------------------------------------------------------
tail_weight: 0.01 #0.01, 0.998
model_sigma: 0.05 #0.003, 0.005
sigma_factor: 0.0014247 #0.0014247
half_life_depth: 1.0
max_depth: 6.0
min_depth: 0.0
# approximate model
fukf_tail_weight: 0.01 # 0.001 # 0.01 #0.01, 0.998
fukf_model_sigma: 0.005 #0.003, 0.005
fukf_sigma_factor: 0.0014247 # 0.0014247








object_names: [
 calib_obj
]

sampling_blocks: [
 [ 0,  1,  2,  3,  4,  5]
]

